File: src/routes/coffeeRoutes.ts
import express from 'express';
import { getCoffees, getCoffeeById, processOrder } from '../controllers/coffeeController';

const router = express.Router();

// Get all coffees
router.get('/', getCoffees);

// Get a single coffee by ID
router.get('/:id', getCoffeeById);

// Process an order
router.post('/orders', processOrder);

export default router;

File: pythonversion/part1/src/routes/coffee_routes.py
from __future__ import annotations

from flask import Blueprint
from controllers.coffee_controller import get_coffees, get_coffee_by_id, process_order

coffee_bp = Blueprint('coffee_part1', __name__)

coffee_bp.get('/') (get_coffees)
coffee_bp.get('/<int:item_id>') (get_coffee_by_id)
coffee_bp.post('/orders') (process_order)

